app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationBarWidget.java

```diff
@@ -90,7 +90,7 @@ public void run() {
         mBackButton.setOnClickListener(new OnClickListener() {
             @Override
             public void onClick(View v) {
-                requestFocusFromTouch();
+                mBackButton.requestFocusFromTouch();
```

Comment:
could you use `v.requestFocusFromTouch()` instead of the variable heare and in all the other buttons? This will avoid some duplicated references, a object with a listener with holds himself